src/components/Header/Content/Main/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,52 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../../../../Context/TodosContext';
+import { Status } from '../../../../../types/Status';
+import { Todo } from '../../../../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+export const TodoList: React.FC = () => {
+  const { todos, status, tempTodo } = useContext(TodosContext);
+
+  let filterTodos = todos;
+
+  const activeFilter = filterTodos.filter((todo: Todo) => !todo.completed);
+  const completedFilter = filterTodos.filter((todo: Todo) => todo.completed);
+
+  switch (status) {
+    case Status.All:
+      filterTodos = todos;
+      break;
+
+    case Status.Active:
+      filterTodos = activeFilter;
+      break;
+
+    case Status.Completed:
+      filterTodos = completedFilter;
+      break;
+
+    default:
+      break;
+  }
```

Comment:
```suggestion
const filteredTodos = todos.filter(({completed}) => {
  switch(status) {
    case Status.Active:
      return: !completed;
      
    case Status.Completed:
      return: completed;
      
    default:
      return true;
  }
})
```

looks better, isn't it?)

Reply:
also it makes sense to use useMemo hook here

Reply:
not fixed